client/src/modules/reports/generate/account_report_multiple/account_report_multiple.config.js

```diff
@@ -0,0 +1,107 @@
+angular.module('bhima.controllers')
+  .controller('account_report_multipleController', AccountReportMultipleConfigController);
+
+AccountReportMultipleConfigController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService', 'AppCache', 'reportData',
+  '$state', 'moment', 'SessionService',
+];
+
+function AccountReportMultipleConfigController(
+  $sce, Notify, SavedReports, AppCache, reportData, $state,
+  Moment, Session
+) {
+  const vm = this;
+  const cache = new AppCache('configure_account_report_multiple');
+  const reportUrl = 'reports/finance/account_report_multiple';
+  vm.previewGenerated = false;
+  vm.accountIds = [];
+  delete cache.reportDetails;
+  vm.reportDetails = {
+    currency_id : Session.enterprise.currency_id,
+    selectedAccounts : [],
```

Comment:
Shouldn't we save the accounts so that when a user comes back to this screen, they find the same accounts?  It's how the other account report works.  See below.

![2018-09-10_13-23-51](https://user-images.githubusercontent.com/896472/45297298-da2de800-b4fc-11e8-9644-0a73e01fbc02.gif)

Reply:
It's already like that